hw07-patterns/src/main/java/ru/otus/edu/levina/atm/impl/cmd/AbstrCmd.java

```diff
@@ -0,0 +1,40 @@
+package ru.otus.edu.levina.atm.impl.cmd;
+
+import java.util.Date;
+import java.util.UUID;
+
+import lombok.Data;
+import ru.otus.edu.levina.atm.api.ATM;
+import ru.otus.edu.levina.atm.api.cmd.CmdResponse;
+import ru.otus.edu.levina.atm.api.cmd.Command;
+
+@Data
+public abstract class AbstrCmd implements Command {
```

Comment:
```suggestion
public abstract class AbstractCommand implements Command {
```
или 

```suggestion
public abstract class BaseCommand implements Command {
```

В java как-то не принято стремиться минимизировать количество символов в именах

`ноТакиеДлинныеИменаТожеНеСтоитПисатьХотяЧастенькоТакоеВстречаетсяВИменахТестов`